### PR TITLE
Fixes #19059: rework 6.2.3/6.2.4 release notes

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/19059

### CODE REVIEW

File: release-data/changelogs/6.1/main.adoc [hunk 1]
```diff
@@ -42,13 +42,43 @@ This is a bug fix release in the 6.1 series and therefore all installations of 6
...
-* https://issues.rudder.io/issues/18955[#18955]: OOM exception now stops Rudder correctly with information logs either in rudder-jetty service or in webapp logs, but some JVM, especially old Java 8, don't log anything
+Previously, when adding a new parameter to a technique in the technique editor, all directives based on this technique
+were still valid and used an empty value for the new parameter. This leads to unexpected behaviors, and makes adding
+parameters to techniques potentially dangerous.
+
+To prevent this type of problems (in the scope of a patch release) we had to totally prevent passing empty parameters to techniques from the technique editor. This makes missing parameters a policy generation error, allowing to safely provide
+them to directives after modifying the source technique.
+
+This may break existing directives that rely on an empty parameter (wether with an empty value or with an expression evaluating to an empty value, like a property containing an empty string).
```
Comment (VinceMacBuche) [inline release-data/changelogs/6.1/main.adoc:54]: you could not set an empty value on this field

Comment (VinceMacBuche) [reply release-data/changelogs/6.1/main.adoc:54]: it was rejected, only the case where you use a property with a default value is broken

**Comment by amousset** [comment]
PR updated with a new commit

File: release-data/changelogs/6.1/main.adoc [hunk 2]
```diff
@@ -42,13 +42,43 @@ This is a bug fix release in the 6.1 series and therefore all installations of 6
...
-* https://issues.rudder.io/issues/18955[#18955]: OOM exception now stops Rudder correctly with information logs either in rudder-jetty service or in webapp logs, but some JVM, especially old Java 8, don't log anything
+Previously, when adding a new parameter to a technique in the technique editor, all directives based on this technique
+were still valid and used an empty value for the new parameter. This leads to unexpected behaviors, and makes adding
+parameters to techniques potentially dangerous.
+
+To prevent this type of problems (in the scope of a patch release) we had to totally prevent passing empty parameters to techniques from the technique editor. This makes missing parameters a policy generation error, allowing to safely provide
+them to directives after modifying the source technique.
+
+This may break existing directives that rely on an expression evaluating to an empty value, like a property containing an empty string, or a property value with an empty default.
```
Comment (VinceMacBuche) [inline release-data/changelogs/6.1/main.adoc:54]: evaluating -> Which evaluates

**Comment by amousset** [comment]
PR updated with a new commit

**Review by VinceMacBuche — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**